workers/unity/Packages/io.improbable.gdk.core/Tests/Editmode/Connection/CommandLineConnectionFlowInitializerTests.cs

```diff
@@ -0,0 +1,148 @@
+using System.Collections.Generic;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.Core.EditmodeTests.Connection
+{
+    [TestFixture]
+    public class CommandLineConnectionFlowInitializerTests
+    {
+        private static Dictionary<string, string> BaseArgs => new Dictionary<string, string>
```

Comment:
I think this might be clearer as a function. Had a double take when I saw the usage below as its not clear `BaseArgs` is a property returning a NEW dictionary every time.

Reply:
Fair point - can refactor it into a small function